js/modules/loki_file_server_api.js

```diff
@@ -219,10 +219,26 @@ class LokiHomeServerInstance extends LokiFileServerInstance {
 
   async updateOurDeviceMapping() {
     const isPrimary = !storage.get('isSecondaryDevice');
-    const authorisations = await window.libsession.Protocols.MultiDeviceProtocol.getPairingAuthorisations(
+    let authorisations = await window.libsession.Protocols.MultiDeviceProtocol.getPairingAuthorisations(
```

Comment:
```suggestion
    const authorisations = await window.libsession.Protocols.MultiDeviceProtocol.getPairingAuthorisations(
```
it'd be better if the mapping below mapped onto a new const variable instead. Just trying to avoid any side effects.